### PR TITLE
Dropdown menu for log out, ratings, and data upload per design spec 

### PR DESCRIPTION
**Pull request by cmruss**
# Description
[TRY ME](https://dropdown-logout.d1ms51d9oybzjy.amplifyapp.com/)
Updates nav layout to the design spec nesting logout, ratings, and data upload to the user icon dropdown menu

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?

- [x] Manual testing from local and deployed
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: src/components/dashboard/navigation.js [hunk 1]
```diff
@@ -126,14 +113,40 @@ class Navigation extends Component {
...
+                to="/login"
+                >
+                  Log out
+               </NavLink>              
+               </div>
+             </div>
+
+             <FontAwesomeIcon className="angle-down-icon hidden" icon={faAngleDown} />
+              <i className="far fa-angle-down"></i>
+
```
Comment (briannakeune) [inline src/components/dashboard/navigation.js]: Why are we hiding the angle down?

Comment (cmruss) [reply src/components/dashboard/navigation.js]: I'm not really sure what it's there for. It is not present in the mobile hifi. 
I could either remove it or conditionally render it based on hover to be open or closed, and present or not on media query..
I vote delete, but I'm open to collaboration

Comment (briannakeune) [reply src/components/dashboard/navigation.js]: I think it's there to let users know that it's a dropdown menu on desktop. I didn't look at the mobile hifi's but I know it's there for desktop. :)

Comment (cmruss) [reply src/components/dashboard/navigation.js]: I have added it to the responsive-navbar branch, which I am requesting to merge into this branch as usability is nonexistent on a smaller screen. It appears and disappears depending on screen size.

**Pull request merged**